editor/project_manager.cpp

```diff
@@ -201,19 +204,19 @@ String ProjectDialog::_test_path() {
 				d->list_dir_end();
 
 				if (!is_folder_empty) {
-					_set_message(TTR("Please choose an empty folder."), MESSAGE_WARNING, INSTALL_PATH);
+					_set_message(TTR("Please choose an empty import folder."), MESSAGE_WARNING, INSTALL_PATH);
 					get_ok_button()->set_disabled(true);
 					return "";
 				}
 
 			} else {
-				_set_message(TTR("Please choose a \"project.godot\" or \".zip\" file."), MESSAGE_ERROR);
+				_set_message(TTR("Please choose a project folder or a \".zip\" file."), MESSAGE_ERROR);
 				install_path_container->hide();
 				get_ok_button()->set_disabled(true);
 				return "";
 			}
 
-		} else if (valid_path.ends_with("zip")) {
+		} else if (is_zip) {
 			_set_message(TTR("This directory already contains a Godot project."), MESSAGE_ERROR, INSTALL_PATH);
```

Comment:
Suggest changing message to: "A zipped Godot project already exists at this location."

I read where is_zip is initialized as: not a directory, ends with ".zip", is a file. So the input path isn't a folder containing a zip, it's a path of an existing zip.

Reply:
That isn't the case though, the case is that the path to install to already contains a project